reactDashboard/src/components/DataCharts/DateRangeSlider.jsx

```diff
@@ -103,6 +103,7 @@ function DateRangeSlider({
   councilAreaDataset = null,
   dateRange = { startDate: 0, endDate: 0 },
   setDateRange,
+  darkmode,
```

Comment:
If the component doesn't use the property, best to not pass it in the first place